tools/loader/ImageClassifier.cpp

```diff
@@ -121,7 +138,12 @@ void loadImagesAndPreprocess(const llvm::cl::list<std::string> &filenames,
     for (unsigned z = 0; z < numChannels; z++) {
       for (unsigned y = 0; y < dims[1]; y++) {
         for (unsigned x = 0; x < dims[0]; x++) {
-          RH.at({n, numChannels - 1 - z, x, y}) = (imageH.at({x, y, z}));
+          if(imageformat != "NHWC") {
```

Comment:
can you explicitly check against NCHW here.

Reply:
Please do not use if-else to distinguish between formats. We may have more than 2 over time. Explicitly check for the formats you want to process and assert if you see an unexpected format.

Reply:
Will do